lib/ddtrace/contrib/rails/action_controller.rb

```diff
@@ -66,6 +67,26 @@ def self.finish_processing(payload)
         rescue StandardError => e
           Datadog::Tracer.log.error(e.message)
         end
+
+        def self.exception_controller?(payload)
+          exception_controller_class = Datadog.configuration[:rails][:exception_controller]
+          controller = payload.fetch(:controller)
+          headers = payload.fetch(:headers)
+
+          # If an exception controller class has been specified,
+          # check if the controller is a kind of the exception controller class.
+          if exception_controller_class.is_a?(Class) || exception_controller_class.is_a?(Module)
+            controller <= exception_controller_class
+          # Else if no exception controller class has been set,
+          # guess whether this is an exception controller from the headers.
+          elsif exception_controller_class.nil?
```

Comment:
it's just "cosmetic", but this is the base case because most of developers don't set `exception_controller`, so would be great having it as a first `if` condition.

Reply:
Alright, I can rearrange this.